packages/runtime/src/api/react.tsx

```diff
@@ -1,157 +1,152 @@
-  TypePolicies,
-} from '@apollo/client'
-import { BatchHttpLink } from '@apollo/client/link/batch-http'
-export { gql } from '@apollo/client'
 import { createContext, ReactNode, useContext } from 'react'
 import { KeyUtils } from 'slate'
 
-import { INTROSPECTION_QUERY } from '../components/utils/queries'
-
 import { Element } from '../state/react-page'
-import { PagePathnameSliceFragment, PagePathnameSliceFragmentDoc } from './generated/graphql'
 import { storeContextDefaultValue } from '../runtimes/react'
 import { introspect } from './introspection'
+import * as MakeswiftApiClient from '../state/makeswift-api-client'
+import type {
+  File,
+  GlobalElement,
+  Page,
+  PagePathnameSlice,
+  Site,
+  Snippet,
+  Swatch,
+  Table,
+  Typography,
+} from './graphql/types'
+import { GraphQLClient } from './graphql/client'
+import { CreateTableRecordMutation } from './graphql/documents'
+import {
+  CreateTableRecordMutationResult,
+  CreateTableRecordMutationVariables,
+} from './graphql/generated/types'
 
-const typePolicies: TypePolicies = {
-  Query: {
-    fields: {
-      swatches(existingData, { args, toReference }) {
-        return (
-          existingData ??
-          args?.ids.map((id: string) => toReference({ __typename: 'Swatch', id }, true))
-        )
-      },
-      file(existingData, { args, toReference }) {
-        return existingData ?? toReference({ __typename: 'File', id: args?.id }, true)
-      },
-      files(existingData, { args, toReference }) {
-        return (
-          existingData ??
-          args?.ids.map((id: string) => toReference({ __typename: 'File', id }, true))
-        )
-      },
-      typographies(existingData, { args, toReference }) {
-        return (
-          existingData ??
-          args?.ids.map((id: string) => toReference({ __typename: 'Typography', id }, true))
-        )
-      },
-      pagePathnamesById(existingData, { args, toReference }) {
-        return (
-          existingData ??
-          args?.ids.map((id: string) => toReference({ __typename: 'PagePathnameSlice', id }, true))
-        )
-      },
-      globalElement(existingData, { args, toReference }) {
-        return existingData ?? toReference({ __typename: 'GlobalElement', id: args?.id }, true)
-      },
-      table(existingData, { args, toReference }) {
-        return existingData ?? toReference({ __typename: 'Table', id: args?.id }, true)
-      },
-      page(existingData, { args, toReference }) {
-        return existingData ?? toReference({ __typename: 'Page', id: args?.id }, true)
-      },
-      site(existingData, { args, toReference }) {
-        return existingData ?? toReference({ __typename: 'Site', id: args?.id }, true)
-      },
-    },
-  },
+export type MakeswiftClientOptions = {
+  uri: string
+  serializedState?: MakeswiftApiClient.SerializedState
 }
 
-type CreateApolloClientParams = {
-  uri?: string
-  cacheData?: NormalizedCacheObject
-}
+// NOTE(miguel): MakeswiftClient -> HostClient
```

Comment:
Expand on this note or remove it. As it is it will probably be confusing in a couple of months.